diff --git a/node_modules/expo-device/ios/UIDevice.swift b/node_modules/expo-device/ios/UIDevice.swift
index 6bfc96a..a0b87e6 100644
--- a/node_modules/expo-device/ios/UIDevice.swift
+++ b/node_modules/expo-device/ios/UIDevice.swift
@@ -185,7 +185,11 @@ public extension UIDevice {
   // Credit: https://github.com/developerinsider/isJailBroken/blob/master/IsJailBroken/Extension/UIDevice%2BJailBroken.swift
   var isSimulator: Bool {
-    return TARGET_OS_SIMULATOR != 0
+#if targetEnvironment(simulator)
+    return true
+#else
+    return false
+#endif
   }
